tests/intensive/sam_parity_tests.py

```
@@ -1694,5 +1791,444 @@ def test_combined_box_and_point_prompt_parity(self):
         )
 
 
+class TestSAM3VideoConceptParity(unittest.TestCase):
+    """Compare FiftyOne SAM3 video concept-mode output against direct
+    concept_predictor (handle_request) inference."""
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.15)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)